### ISSUE TITLE
[WinUI 3 Gallery: Design->Color]: Color contrast ratio for 'Copy brush name' (copy icon) button is 2.434:1 which is less than 3:1

### ISSUE THREAD

**Issue by sabareesanms** [microsoft/WinUI-Gallery#1948]
**Environment Details:**

**OS Version:** 24H2 (OS Build 26100.4315)

**App:** WinUI 3 Gallery

**App Version** : 2.6.0.0 (SDK 1.7)

**AT:** Color Contrast

**Repro Steps:**

1. Launch the WinUI 3 Gallery app.

2. Navigate to search box using tab key.

3. Type 'Color' in search box and hit enter.

4. Navigate to 'Color' card and activate it.

5. Now check color contrast ratio for 'Copy brush name' (icon) button present beside the control strong / disabled text

**Actual Result:**

Color contrast ratio for 'Copy brush name' (copy icon) button is 2.434:1 which is less than 3:1

**Expected Result:**

Color contrast for  'Copy brush name' (copy icon) button should be equal to or greater than 3:1.

**User impact:**

Low vision Users will face difficulty in Understanding and accessing the Icon button, if Icon is not reaching the required contrast ratio.
